Assets/Scripts/AdManager.cs

```diff
@@ -18,9 +17,10 @@ public class AdManager : MonoBehaviour {
     [SerializeField]
     private Toggle offerwallCloseOnFirstClickToggle;
     [SerializeField]
-    private Text offerwallCurrencyIdTextField;
+    private readonly Text offerwallCurrencyIdTextField;
```

Comment:
Why this change? Does this not affect the setting of the currency id on the testapp?

Reply:
I am not sure. I could remove it and we would be safer, if that's fine.

Reply:
Did you test it? If so, then we're good. Otherwise I wouldn't change it